components/ord-service/src/main/java/com/sap/cloud/cmp/ord/service/storage/model/PackageEntity.java

```diff
@@ -66,6 +66,9 @@
     @Column(name = "licence_type", length = 256)
     private String licenseType;
 
+    @Column(name = "support_info")
```

Comment:
The default length is 255, you need to explicitly set it to length = Integer.MAX_VALUE